templates/templates/_navigation.html

```diff
@@ -2,204 +2,202 @@
-                        <a class="p-navigation__link"
-                           href="/navigation#get-ubuntu-navigation"
-                           aria-controls="#get-ubuntu-content"
-                           tabindex="0"
-                           onfocus="fetchDropdown('/templates/meganav/get-ubuntu', 'get-ubuntu');">Get Ubuntu</a>
-                    </li>
-                    <li class="p-navigation__item--dropdown-toggle global-nav-mobile global-nav"
-                        role="menuitem"
-                        id="all-canonical"></li>
-                    <li class="p-navigation__item--dropdown-toggle js-account"
-                        role="menuitem"
-                        id="canonical-login"></li>
-                    <li class="p-navigation__item">
-                        <a href="/search"
-                           class="js-search-button p-navigation__link--search-toggle"></a>
-                    </li>
-                </ul>
-                <div class="p-navigation__search">
-                    <form action="/search" class="p-search-box is-light">
-                        <input type="search"
-                               class="p-search-box__input"
-                               name="q"
-                               placeholder="Search our sites"
-                               required=""
-                               aria-label="Search our sites" />
-                        <button type="reset" class="p-search-box__reset">
-                            <i class="p-icon--close"></i>
-                        </button>
-                        <button type="submit" class="p-search-box__button">
-                            <i class="p-icon--search"></i>
-                        </button>
-                    </form>
-                </div>
-            </nav>
+  {% block header_banner %}
+    <div class="p-navigation__row--25-75">
+      <div class="p-navigation__banner">
+        <div class="p-navigation__tagged-logo">
+          <a class="p-navigation__link" href="/">
+            {% if breadcrumbs and breadcrumbs.children or breadcrumbs.grandchildren %}
+              Canonical Ubuntu
+            {% else %}
+              <div class="p-navigation__logo-tag">
+                <img class="p-navigation__logo-icon"
+                     src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg"
+                     alt="" />
+              </div>
+              <span class="p-navigation__logo-title">Canonical Ubuntu</span>
+            {% endif %}
+          </a>
+        </div>
+        <ul class="p-navigation__items">
+          <li class="p-navigation__item">
+            <a href="/search"
+               class="js-search-button p-navigation__link--search-toggle"
+               aria-label="Search"></a>
+          </li>
+          <li class="p-navigation__item">
+            <a href="#" class="js-menu-button p-navigation__link">Menu</a>
```

Comment:
Don't you want to keep the `/navigation` for no JS version?

Reply:
good catch